HelpSource/Classes/MIDIdef.schelp

```diff
@@ -6,8 +6,12 @@ related:: Guides/MIDI, Classes/MIDIFunc
 DESCRIPTION::
 MIDIdef provides a global reference to the functionality of its superclass link::Classes/MIDIFunc::. Essentially it stores itself at a key within a global dictionary, allowing replacement at any time. Most methods are inherited from its superclass.
 
-note:: MIDIdef and link::Classes/MIDIFunc:: aim to improve upon the MIDIresponder classes by being faster, easier to use, and providing support for all MIDI message types. They were made with the intention of creating a more convenient, logical and consistent interface, which shares a common design with link::Classes/OSCFunc:: and link::Classes/OSCdef::. Note that unlike the older classes, MIDIdefs are removed on Cmd-. by default. This can be overridden using either of the fix or permanent methods.::
 
+subsection:: Important note on persistence
+
+MIDIFunc (and its subclass link::Classes/MIDIdef::) are by default removed when the user presses Cmd/ctrl-. to stop SuperCollider.
```

Comment:
it is a little misleading to say that Cmd/ctrl-. to stops SuperCollider. 
Suggestion:

MIDIFunc (and its subclass link::Classes/MIDIdef:: like all other AbstractResponderFuncs) are by default removed on pressing Cmd/ctrl-.